bin/extract-translations.sh

```diff
@@ -11,6 +11,6 @@ echo '# Extracting translations from vendor/ezsystems/hybrid-platform-ui';
 cd vendor/ezsystems/hybrid-platform-ui;
 
 echo '# Clean file references';
-sed -i .bak "s|>.*/hybrid-platform-ui/|>|g" ./src/bundle/Resources/translations/*.xlf
+sed -i "s|>.*/hybrid-platform-ui/|>|g" ./src/bundle/Resources/translations/*.xlf
```

Comment:
it's because there a small difference in the `-i` option between BSD sed (the one on Mac) and GNU sed (usually the one on Linux). To have something which works well in both case, this line should be written:

```shell
sed -i.bak "s|>.*/hybrid-platform-ui/|>|g" ./src/bundle/Resources/translations/*.xlf
```

(no space between `-i` and `.bak`)